tools/README.md

```diff
@@ -116,19 +116,14 @@ The SonarQube server instance is created with standard insecure credentials (Use
 
 The BCDevOps SonarQube repo provides a script that will generate a random PW, set it in SonarQube, and create a secret. This can be found under the  `/provisioning` folder of the cloned BCDevOps repo.
 
-In order to directly get the password reset script, run the following:
+In order to directly get and automatically run the password reset script, execute the following:
 
 ```sh
-curl https://raw.githubusercontent.com/BCDevOps/sonarqube/bbb9f62e29706b61382cf24d7ad7e08f2476a01f/provisioning/updatesqadminpw.sh > updatesqadminpw.sh
+curl https://raw.githubusercontent.com/BCDevOps/sonarqube/bbb9f62e29706b61382cf24d7ad7e08f2476a01f/provisioning/updatesqadminpw.sh | sh
 ```
+This will download and run the password reset script and save a new random password in the secrets manager under ``sonarqube-admin-password``
 
-Then simply run the following script and follow its instructions. Make sure you save the new password in an OpenShift secret or equivalent!  Ensure that the SonarQube app is fully deployed and operational before running this script.
-
-```sh
-chmod +x updatesqadminpw.sh && ./updatesqadminpw.sh
-```
```

Comment:
If I recall correctly, this command outputs to file deliberately in order to maintain Windows compatibility. I definitely agree that your change makes sense in a Unix context as we can just pipe and stream the contents - however it would only work on unix based systems/environments.

Reply:
If using gitbash or WSL, this would in fact work ok (I am using the former on Windows). Maybe a better option to the command would be to provide quick copy/paste for osx/unix environments, and a windows-based one?  That way those who know their environments well could pick and choose which version they want.

Reply:
I've just re-ran the new command and it appears to work on git bash/mingw64 now. A few versions ago this command wouldn't have worked for piping so it should good. Please restore the `Ensure that the SonarQube app is fully deployed and operational before running this script.` in some form or another to remind the user not to run the password reset script until the pod is up and ready and we can merge this in.